File: Lab1_2/C++/main.cpp
#include <iostream>
#include "Class.h"

int main() {
    // Конструктор без параметрів
    StringClass s1;

    // Конструктор з параметром
    StringClass s2("ello VladislavH");

    // Конструктор копіювання
    StringClass s3(s2);

	// Використання методів
    std::cout << "Text: " << s2.GetText() << std::endl;
    std::cout << "Length: " << s2.Length() << std::endl;
	// Зсув рядка
    s2.ShiftRight();
    std::cout << "After shift: " << s2.GetText() << std::endl;

    return 0;
}

File: Lab1_2/C#/main.cs
using System;
using StringLibrary;

class MainClass
{
    static void Main()
    {
        // Конструктор без параметрів
        StringClass s1 = new StringClass();

        // Конструктор з параметром
        StringClass s2 = new StringClass("ello VladislavH");

        // Конструктор копіювання
        StringClass s3 = new StringClass(s2);
        // Використання методів
        Console.WriteLine("Text: " + s2.GetText());
        Console.WriteLine("Length: " + s2.Length());
        // Зсув
        s2.ShiftRight();
        Console.WriteLine("After shift: " + s2.GetText());
    }
}